rig/machine_control/regions.py

```diff
@@ -0,0 +1,200 @@
+"""SpiNNaker machine regions.
+
+.. todo::
+    Neaten this documentation!
+
+Regions (not to be confused with rig Regions) are used to specify areas of a
+SpiNNaker machine for the purposes of transmitting nearest neighbour packets or
+for determining which chips should be included in any flood-fill of data or
+application loading.
+
```

Comment:
Update from ST: "No, it was really a discussion document and just circulated by email. The current implementation has made it slightly inaccurate." I have suggested making the region part a standalone appnote and ST is going to look into doing this.

For the moment the above ref is going to be the best option.